[2025-03-29T14:58:19.327+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-29T14:58:19.387+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fist_dag_v5.first_task scheduled__2025-03-07T00:00:00+00:00 [queued]>
[2025-03-29T14:58:19.413+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fist_dag_v5.first_task scheduled__2025-03-07T00:00:00+00:00 [queued]>
[2025-03-29T14:58:19.422+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2025-03-29T14:58:19.470+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): first_task> on 2025-03-07 00:00:00+00:00
[2025-03-29T14:58:19.488+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1893) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-29T14:58:19.493+0000] {standard_task_runner.py:72} INFO - Started process 1905 to run task
[2025-03-29T14:58:19.488+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fist_dag_v5', 'first_task', 'scheduled__2025-03-07T00:00:00+00:00', '--job-id', '608', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpscp4033b']
[2025-03-29T14:58:19.496+0000] {standard_task_runner.py:105} INFO - Job 608: Subtask first_task
[2025-03-29T14:58:19.609+0000] {task_command.py:467} INFO - Running <TaskInstance: fist_dag_v5.first_task scheduled__2025-03-07T00:00:00+00:00 [running]> on host ba06481f48a9
[2025-03-29T14:58:19.923+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='fist_dag_v5' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-07T00:00:00+00:00'
[2025-03-29T14:58:19.925+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-29T14:58:19.966+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-29T14:58:19.969+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task!']
[2025-03-29T14:58:19.990+0000] {subprocess.py:99} INFO - Output:
[2025-03-29T14:58:19.993+0000] {subprocess.py:106} INFO - hello world, this is the first task!
[2025-03-29T14:58:19.994+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-29T14:58:20.045+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-29T14:58:20.047+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=fist_dag_v5, task_id=first_task, run_id=scheduled__2025-03-07T00:00:00+00:00, execution_date=20250307T000000, start_date=20250329T145819, end_date=20250329T145820
[2025-03-29T14:58:20.129+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-29T14:58:20.202+0000] {taskinstance.py:3895} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2025-03-29T14:58:20.212+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
